packages/remix-react/components.tsx

```diff
@@ -1309,3 +1310,18 @@ function useComposedRefs<RefValueType = any>(
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, refs);
 }
+
+let OutletContext = React.createContext<AppData>(null);
+
+export function Outlet<Data = AppData>({ context }: { context?: Data }) {
+  return (
+    <OutletContext.Provider value={context}>
```

Comment:
What if it's called data? The generic is also named that way
```suggestion
export function Outlet<Data = AppData>({ data }: { data?: Data }) {
  return (
    <OutletContext.Provider value={data}>
```

Reply:
It's not necessarily data, it's just ... whatever. Could be a function or a Symbol!